# HG changeset patch
# User John W. Eaton <jwe@octave.org>
# Date 1436846519 14400
#      Tue Jul 14 00:01:59 2015 -0400
# Node ID 0e2da885a410c209233ab6473df19e78623c184d
# Parent  7ddee729f29ee7cb7d82d120fde42129068f3f7d
fix lists of files to distribute

* Makefile.am, doc/module.mk, examples/module.mk: Include more files
in lists of files to distribute.

diff --git a/Makefile.am b/Makefile.am
--- a/Makefile.am
+++ b/Makefile.am
@@ -22,16 +22,18 @@ include build-aux/common.mk
 
 image_DATA =
 octdata_DATA =
 octetc_DATA =
 octlocale_DATA =
 
 ACLOCAL_AMFLAGS = -I m4
 
+DOC_TARGETS =
+
 BUILT_DISTFILES =
 BUILT_NODISTFILES =
 EXTRA_DIST =
 
 BUILT_DISTFILES += \
   AUTHORS \
   BUGS \
   ChangeLog \
diff --git a/doc/module.mk b/doc/module.mk
--- a/doc/module.mk
+++ b/doc/module.mk
@@ -23,19 +23,16 @@ am__v_PDFTEX_1 =
 
 if AMCOND_BUILD_DOCS
 
 ## Listing "dvi" explicitly here seems to cause two simultaneous
 ## builds of the DVI files, presumably because the PS version
 ## also depends on the DVI file and somehow the rules are invoked
 ## twice.  Is that a bug in automake or make or what?
 
-DOC_TARGETS = \
-  doc/interpreter/doc-cache
-
 EXTRA_DIST += \
   doc/texinfo.tex \
   doc/texmf.cnf
 
 EXTRA_DIST += \
   doc/doxyhtml/Doxyfile.in \
   doc/doxyhtml/README
 
@@ -263,21 +260,33 @@ doc/interpreter/octave.pdf: doc/interpre
 	  rm -rf $(OCTAVE_HTML_TMP_DIR); exit 1; \
 	fi
 
 $(HTMLDIR_IMAGES) : doc/interpreter/octave.html/%.png: doc/interpreter/%.png $(OCTAVE_HTML_STAMP)
 	$(AM_V_GEN)cp $< $@
 
 DOC_TARGETS += \
   $(srcdir)/doc/interpreter/octave.info \
+  doc/interpreter/doc-cache \
   doc/interpreter/octave.ps \
   doc/interpreter/octave.pdf \
   $(OCTAVE_HTML_STAMP) \
   $(HTMLDIR_IMAGES)
 
+EXTRA_DIST += \
+  $(BUILT_OCTAVE_TEXI_SRC) \
+  $(srcdir)/doc/interpreter/octave.info \
+  doc/interpreter/TODO \
+  doc/interpreter/doc-cache \
+  doc/interpreter/octave.dvi \
+  doc/interpreter/octave.ps \
+  doc/interpreter/octave.pdf \
+  doc/interpreter/octave.html \
+  $(HTMLDIR_IMAGES)
+
 ## The texi2dvi script (used to create both PDF and DVI output formats)
 ## uses some fixed temporary file names.  In order to avoid a race condition
 ## the DVI and PDF builds are forced to run serially through a Makefile rule.
 #doc/interpreter/octave.pdf: doc/interpreter/octave.dvi
 
 # Prevent packaging of distribution unless all libraries
 # necessary to create documentation are present
 doc-interpreter-dist-hook:
@@ -390,16 +399,24 @@ info_TEXINFOS += \
   doc/liboctave/liboctave.texi
 
 DOC_TARGETS += \
   $(srcdir)/doc/liboctave/liboctave.info \
   doc/liboctave/liboctave.ps \
   doc/liboctave/liboctave.pdf \
   doc/liboctave/liboctave.html
 
+EXTRA_DIST += \
+  $(liboctave_TEXINFOS) \
+  $(srcdir)/doc/liboctave/liboctave.info \
+  doc/liboctave/liboctave.dvi \
+  doc/liboctave/liboctave.ps \
+  doc/liboctave/liboctave.pdf \
+  doc/liboctave/liboctave.html
+
 ## The texi2dvi script (used to create both PDF and DVI output formats)
 ## uses some fixed temporary file names.  In order to avoid a race condition
 ## the DVI and PDF builds are forced to run serially through a Makefile rule.
 #doc/liboctave/liboctave.pdf: doc/liboctave/liboctave.dvi
 
 liboctave-clean:
 	rm -rf doc/liboctave/t2d_cache
 
@@ -445,18 +462,18 @@ doc/refcard/refcard-legal.ps: doc/refcar
 	-$(AM_V_DVIPS)$(DVIPS) $(AM_V_texinfo) -T 14in,8.5in -o $@ $<
 
 doc/refcard/refcard-letter.pdf: doc/refcard/refcard.tex
 doc/refcard/refcard-letter.dvi: doc/refcard/refcard.tex
 doc/refcard/refcard-letter.ps: doc/refcard/refcard-letter.dvi
 	-$(AM_V_DVIPS)$(DVIPS) $(AM_V_texinfo) -T 11in,8.5in -o $@ $<
 
 EXTRA_DIST += \
-  $(refcard_TEX_SRC) \
-  $(refcard_FORMATTED)
+  $(refcard_FORMATTED) \
+  $(refcard_TEX_SRC)
 
 CLEANFILES += \
   doc/refcard/refcard-a4.log \
   doc/refcard/refcard-legal.log \
   doc/refcard/refcard-letter.log
 
 MAINTAINERCLEANFILES += \
   $(refcard_FORMATTED)
diff --git a/examples/module.mk b/examples/module.mk
--- a/examples/module.mk
+++ b/examples/module.mk
@@ -53,9 +53,10 @@ examples_code_SRC = \
   examples/code/standalonebuiltin.cc \
   examples/code/stringdemo.cc \
   examples/code/structdemo.cc \
   examples/code/unwinddemo.cc
 
 EXTRA_DIST += \
   $(examples_data_SRC) \
   $(examples_code_SRC) \
+  examples/code/COPYING \
   examples/module.mk
